{"uid":"4aedfd5e5c27ff5d","name":"test_landscape_scalable_bundle_db","fullName":"tests.integration.test_db_admin#test_landscape_scalable_bundle_db","historyId":"59582530be6d9d50d6da4914c77954ea","time":{"start":1750470241896,"stop":1750472483928,"duration":2242032},"description":"Deploy Landscape Scalable Bundle to test the 'db-admin' relation.","descriptionHtml":"<p>Deploy Landscape Scalable Bundle to test the 'db-admin' relation.</p>\n","status":"failed","statusMessage":"AssertionError","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_db_admin.py\", line 133, in test_landscape_scalable_bundle_db\n    switchover(ops_test, primary, patroni_password)\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1069, in switchover\n    for attempt in Retrying(stop=stop_after_attempt(30), wait=wait_fixed(2), reraise=True):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 445, in __iter__\n    do = self.iter(retry_state=retry_state)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 378, in iter\n    result = action(retry_state)\n             ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 420, in exc_check\n    raise retry_exc.reraise()\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 187, in reraise\n    raise self.last_attempt.result()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/concurrent/futures/_base.py\", line 449, in result\n    return self.__get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/concurrent/futures/_base.py\", line 401, in __get_result\n    raise self._exception\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1076, in switchover\n    assert standbys == len(ops_test.model.applications[app_name].units) - 1\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nAssertionError","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1750470241724,"stop":1750470241725,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1750470241725,"stop":1750470241725,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1750470241725,"stop":1750470241725,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1750470241725,"stop":1750470241895,"duration":170},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1750470241895,"stop":1750470241895,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy Landscape Scalable Bundle to test the 'db-admin' relation.","status":"failed","statusMessage":"AssertionError","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_db_admin.py\", line 133, in test_landscape_scalable_bundle_db\n    switchover(ops_test, primary, patroni_password)\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1069, in switchover\n    for attempt in Retrying(stop=stop_after_attempt(30), wait=wait_fixed(2), reraise=True):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 445, in __iter__\n    do = self.iter(retry_state=retry_state)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 378, in iter\n    result = action(retry_state)\n             ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 420, in exc_check\n    raise retry_exc.reraise()\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/tenacity/__init__.py\", line 187, in reraise\n    raise self.last_attempt.result()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/concurrent/futures/_base.py\", line 449, in result\n    return self.__get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/concurrent/futures/_base.py\", line 401, in __get_result\n    raise self._exception\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1076, in switchover\n    assert standbys == len(ops_test.model.applications[app_name].units) - 1\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nAssertionError","steps":[],"attachments":[{"uid":"598bfd91526c62ff","name":"log","source":"598bfd91526c62ff.txt","type":"text/plain","size":31445}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1750472483941,"stop":1750472483941,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1750472483943,"stop":1750472484872,"duration":929},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1750472484873,"stop":1750472484873,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1750472484873,"stop":1750472484873,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1750472484873,"stop":1750472484873,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_db_admin"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"15450-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_db_admin"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c2d2b13915b8fc7b","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[]}],"history":{"statistic":{"failed":6,"broken":94,"skipped":31,"passed":325,"unknown":16,"total":472},"items":[{"uid":"6d7c35216e5ac506","reportUrl":"../5017//#testresult/6d7c35216e5ac506","status":"passed","time":{"start":1750383588646,"stop":1750385322168,"duration":1733522}},{"uid":"72d625751f4ae6cc","reportUrl":"../5011//#testresult/72d625751f4ae6cc","status":"passed","time":{"start":1750297538497,"stop":1750300398755,"duration":2860258}},{"uid":"ecbe056bfe0bdee","reportUrl":"../5008//#testresult/ecbe056bfe0bdee","status":"passed","time":{"start":1750210529042,"stop":1750212328787,"duration":1799745}},{"uid":"d09f82157fd10988","reportUrl":"../4994//#testresult/d09f82157fd10988","status":"passed","time":{"start":1750124553272,"stop":1750126290895,"duration":1737623}},{"uid":"9ea6132eb7467e98","reportUrl":"../4984//#testresult/9ea6132eb7467e98","status":"passed","time":{"start":1750037811139,"stop":1750040065460,"duration":2254321}},{"uid":"266bcc2ff185058b","reportUrl":"../4983//#testresult/266bcc2ff185058b","status":"passed","time":{"start":1749951594813,"stop":1749953442142,"duration":1847329}},{"uid":"3ca1cb55bc428edc","reportUrl":"../4982//#testresult/3ca1cb55bc428edc","status":"passed","time":{"start":1749864931306,"stop":1749866753751,"duration":1822445}},{"uid":"bc2099819b41aace","reportUrl":"../4973//#testresult/bc2099819b41aace","status":"passed","time":{"start":1749778804809,"stop":1749781179648,"duration":2374839}},{"uid":"69fdba136034732e","reportUrl":"../4966//#testresult/69fdba136034732e","status":"passed","time":{"start":1749692074153,"stop":1749694018927,"duration":1944774}},{"uid":"6e9227ec0dc1de3f","reportUrl":"../4948//#testresult/6e9227ec0dc1de3f","status":"passed","time":{"start":1749605799088,"stop":1749607868349,"duration":2069261}},{"uid":"e568d953992f547e","reportUrl":"../4927//#testresult/e568d953992f547e","status":"passed","time":{"start":1749520161557,"stop":1749522361221,"duration":2199664}},{"uid":"eaaba5527b05497b","reportUrl":"../4900//#testresult/eaaba5527b05497b","status":"passed","time":{"start":1749433203994,"stop":1749435321666,"duration":2117672}},{"uid":"32746c1fc72a67fb","reportUrl":"../4895//#testresult/32746c1fc72a67fb","status":"failed","statusDetails":"AssertionError","time":{"start":1749346571809,"stop":1749347910827,"duration":1339018}},{"uid":"d8f2d15da17d312e","reportUrl":"../4892//#testresult/d8f2d15da17d312e","status":"passed","time":{"start":1749260258769,"stop":1749262152867,"duration":1894098}},{"uid":"83a3d76a7d88f2f6","reportUrl":"../4869//#testresult/83a3d76a7d88f2f6","status":"passed","time":{"start":1749173946343,"stop":1749176263451,"duration":2317108}},{"uid":"63c15ec5628deaca","reportUrl":"../4840//#testresult/63c15ec5628deaca","status":"passed","time":{"start":1749087484931,"stop":1749089690654,"duration":2205723}},{"uid":"33dda790a1e1f53b","reportUrl":"../4817//#testresult/33dda790a1e1f53b","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1749000866726,"stop":1749001671981,"duration":805255}},{"uid":"16803055734931e1","reportUrl":"../4789//#testresult/16803055734931e1","status":"failed","statusDetails":"AssertionError","time":{"start":1748914945304,"stop":1748917561100,"duration":2615796}},{"uid":"11be1a0a450b86","reportUrl":"../4758//#testresult/11be1a0a450b86","status":"passed","time":{"start":1748828192990,"stop":1748830286024,"duration":2093034}},{"uid":"a46db37eef3eaefc","reportUrl":"../4753//#testresult/a46db37eef3eaefc","status":"passed","time":{"start":1748742229446,"stop":1748744012535,"duration":1783089}}]},"tags":["asyncio"]},"source":"4aedfd5e5c27ff5d.json","parameterValues":[]}